qiskit_nature/properties/vibrational/__init__.py

```diff
@@ -0,0 +1,56 @@
+# This code is part of Qiskit.
```

Comment:
`vibrational` properties -> `vibrational_structure` properties (same as in the electronic structure case)